muspinsim/tests/test_hamiltonian.py

```diff
@@ -47,6 +47,88 @@ def test_evolve(self):
 
         self.assertTrue(np.all(np.isclose(evol[:, 0], 0.5 * np.cos(2 * np.pi * t))))
 
+    def test_evolve_density(self):
+
+        ssys = SpinSystem(["e"])
+        ssys.add_linear_term(0, [1, 0, 0])  # Precession around x
+        H = ssys.hamiltonian
+        rho0 = DensityOperator.from_vectors()  # Start along z
+        t = np.linspace(0, 1, 4)
+
+        self.assertTrue(isinstance(H, Hamiltonian))
+
+        # No operators => return a list of density operators
+        evol = H.evolve(rho0, t)
+
+        print(evol[0].matrix)
+        print(evol[1].matrix)
+        print(evol[2].matrix)
+        print(evol[3].matrix)
```

Comment:
```suggestion
```
Are these intentional? We're not printing statements for any of the other tests.

Reply:
🤦‍♂️ only used them to get the values for the tests